pkg/virt-operator/kubevirt_test.go

```diff
@@ -2555,6 +2555,44 @@ var _ = Describe("KubeVirt Operator", func() {
 		}, 15)
 	})
 
+	Context("when the monitor namespace does not exist", func() {
```

Comment:
this test looks like it was added in this commit and not moved. Can it be that the original one was not deleted?

Reply:
Thanks for the catch, will have a look!

Reply:
OK, I removed the duplicate test.